Leanplum-SDK/Classes/Features/Variables/LPVarCache.m

```diff
@@ -559,11 +559,11 @@ - (void)applyVariableDiffs:(NSDictionary *)diffs_
                 self.updateBlock();
             }
             
-            if (interfaceUpdated) {
+            if (interfaceUpdated && self.interfaceUpdateBlock) {
```

Comment:
can you explain what this interfaceUpdateBlock check is?

Reply:
I've stumbled upon this and decided to add the null check, since rest of the blocks are guarded also.